package.json

```diff
@@ -32,8 +31,11 @@
     "build:necessary": "lerna run build --scope=@deephaven/icons",
     "analyze": "lerna run analyze",
     "preview": "lerna run --scope=@deephaven/{code-studio,embed-chart,embed-grid} preview --stream",
+    "preview:app": "lerna run --scope=@deephaven/code-studio preview --stream",
     "prestart": "npm run build:necessary",
     "start": "run-p watch:types start:*",
+    "start:e2e": "run-p start:docker start:app",
+    "start:docker": "docker run --detach --publish 10000:10000 ghcr.io/deephaven/server:latest",
```

Comment:
I added this into the GitHub action. I removed these from the npm startup scripts, since they're not actually used (and they conflicted with normal `npm start` functionality).